apps/css-workshop/src/components/Menu.astro

```diff
@@ -0,0 +1,68 @@
+---
+type Props = {
+  isActive?: boolean;
+  checked?: boolean;
+  isLarge?: boolean;
+  disabled?: boolean;
+  isSkeleton?: boolean;
+  iconStyle?: string;
+  iconClassName?: string;
```

Comment:
Instead of `iconStyle` and `iconClassName`, you can have a single prop that does both. And it should be named appropriately, as `startIconProps`.

```suggestion
  startIconProps?: astroHTML.JSX.HTMLAttributes;
```

There should probably also be `endIconProps`.

Reply:
I have combined the props into the `startIconProps` and `endIconProps`.

Reply:
I don't see `startIconProps` and `endIconProps` in your code. I see `iconProps` but it is not even used fully (only `iconProps.class` and `iconProps.style` are being used).

It needs to be spread like this:

```jsx
<span
  aria-hidden='true'
  {...startIconProps}
  class:list={['iui-list-item-icon', startIconProps?.class]}
>
```